src/system/api/rqidApi.ts

```diff
@@ -95,16 +96,17 @@ export class Rqid {
    * **world**: only search in world,
    * **compendiums**: only search in compendiums
    */
-  public static async fromRqidRegex(
+  public static async fromRqidRegexAll(
     rqidRegex: RegExp | undefined,
     rqidDocumentType: string, // like "i", "a", "je"
     lang: string = "en",
     scope: "match" | "all" | "world" | "compendiums" = "match"
+    // onlyFindBest: boolean = false
   ): Promise<Document<any, any>[]> {
     if (!rqidRegex) {
       return [];
     }
-    const result: Document<any, any>[] = [];
+    let result: Document<any, any>[] = [];
```

Comment:
should be const since I'm not reassigning the variable.

Reply:
changed